src/test/java/com/training/assignments/Assignment1Test.java

```diff
@@ -0,0 +1,54 @@
+package com.training.assignments;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class Assignment1Test {
+    Assignment1 assign1 = new Assignment1();
+
+    @Test
+    public void intToDoubleTest()
+    {
```

Comment:
use this curly bracket at line number 12,similarly apply for below methods as well.